modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -65,23 +65,27 @@ $ oc adm upgrade --to-multi-arch
 $ oc adm upgrade
 ----
 +
+.Example output
+
+[source,terminal]
+----
+working towards ${VERSION}: 106 of 841 done (12% complete), waiting on machine-config
+----
++
 [IMPORTANT]
 ====
 Machine launches may fail as the cluster settles into the new state. To notice and recover when machines fail to launch, we recommend deploying machine health checks. For more information about machine health checks and how to deploy them, see _About machine health checks_.
 ====
 +
-//commenting this section out until https://issues.redhat.com/browse/OCPBUGS-8256 is resolved:
-//For `oc get co`, expect `AVAILABLE=True`, `PROGRESSING=False`, and `DEGRADED=False` on all cluster Operators.
-+
-//For `oc get mcp`, expect `UPDATED=True`, `UPDATING=False`, and `DEGRADED=False` on all machine config pools.
+* If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:
```

Comment:
Lastly, the tech preview aspect:

1. Up to you, but maybe adding a link to the page you mention below in the additional resources for this page would be helpful.
2. I am not sure, but is this an appropriate way to inform users that a command is technology preview (i.e. just mentioning technology preview as an adjective in the step)? I would personally expect to see the tech preview snippet below this step and not have TP mentioned in the step at all, but maybe there are examples of this kind of use case
```suggestion
* If you want more detailed information about the status of your update, you can also monitor the migration by running the following command:
```